docs/fuzz-settings.md

```diff
@@ -1,137 +1,356 @@
+fuzz target with each file as input once and exit. This can be used to manually
+reproduce previously found issues.
 
-**Example invocation:**
+**CLI:** Default: `[]`
+
+This is a positional argument that should be specified without a flag as the
+last argument. Here is an example how to specify two corpus directories
+`corpus_dir` and `only_on_startup_corpus` in CLI mode.
 
 ```shell
-npx jazzer target corpus_dir other_corpus
+npx jazzer fuzz-file corpus_dir only_on_startup_corpus
 ```
 
-## Reproducing errors
+It is also possible to provide inputs directly. In that case, Jazzer.js will run
+them once and exit. To run the fuzz target with inputs `crash-abcdef0123456789`
+and `my-test.txt` in CLI can be done as follows:
 
-Once Jazzer.js finds a problematic input, it stores it in the current working
-directory using a problem prefix like `crash-`, `mem-`, `timeout-` or the like.
+```shell
+npx jazzer fuzz-file crash-abcdef0123456789 my-test.txt
+```
 
-This input can then be used to reproduce the issue by specifying it as last
-parameter in the CLI call:
+If no corpus directory is provided, Jazzer.js will start fuzzing from scratch
+and will not save any inputs other than crashes on exit.
 
-```shell
-npx jazzer target crash-abcdef0123456789
+**Jest:** In Jest runner mode, Jazzer.js automatically chooses the corpus
+directories based on the name of the fuzz test and the
+[mode](#mode--fuzzingregression) ("fuzzing" or "regression").
+
+In _regression mode_, the main corpus directory is the directory where the test
+file resides. Each fuzz test has a dedicated corpus subdirectory in it derived
+from its name and the names of the enclosing describe-blocks.
+
+For example, suppose that we have a project with following structure:
+
+```text
+.
+├── package.json
+├── package-lock.json
+├── src
+│   └── ...
+└── tests
+    └── tests.fuzz.js
 ```
 
-## Value profile
+and two fuzz tests in `./tests/tests.fuzz.js`:
 
-Jazzer.js provides coverage and comparison feedback to the internally used
-libFuzzer instance. By setting the libFuzzer flag `-use_value_profile=1` via the
-CLI, new values in intercepted compares are treated as new coverage. This has
-the potential to discover many additional inputs, which would not be detected
-otherwise, but may reduce runtime performance significantly.
+```javascript
+describe("Target", () => {
+	test.fuzz("fuzz test 1", (data) => {});
+	test.fuzz("fuzz test 2", (data) => {});
+});
+```
 
-An example of using value profiling can be found at
-[tests/value_profiling/fuzz.js](../tests/value_profiling/fuzz.js).
+The first time when our example fuzz tests are executed by Jazzer.js by e.g.
+running `npx jest tests/tests.fuzz.js` from the command line, Jazzer.js will
+create the directories `./tests/tests.fuzz/Target/fuzz_test_1` and
+`./tests/tests.fuzz/Target/fuzz_test_2` that the two fuzz tests will use in
+regression mode.
+
+In _fuzzing mode_, the main corpus directory is in the `.cifuzz-corpus/`
```

Comment:
From Jazzer.js' point of view the directory containing crashes is still the /main/ one, the one in `.cifuzz-corpus` is only used for speedups of subsequent runs.

Reply:
Removed "main".